BaseTools/Plugin/HostBasedUnitTestRunner/HostBasedUnitTestRunner.py

```diff
@@ -236,13 +236,17 @@ def gen_code_coverage_clang(self, thebuilder):
         logging.info("Generating UnitTest code coverage")
 
         buildOutputBase = thebuilder.env.GetValue("BUILD_OUTPUT_BASE")
+        workspace = thebuilder.env.GetValue("WORKSPACE")
         if GetHostInfo().os.upper() == "LINUX":
             # Collect test executables with no file extension
             testList = glob.glob(os.path.join(buildOutputBase, "**", "*Test*"), recursive=True)
             testList = [f for f in testList if os.path.isfile(f) and os.path.splitext(f)[1] == ""]
+            allTestList = glob.glob(os.path.join(workspace, "**", "*Test*"), recursive=True)
+            allTestList = [f for f in allTestList if os.path.isfile(f) and os.path.splitext(f)[1] == ""]

```

Comment:
Searching entire workspace for unit test executables is different behavior than gcc or msvc coverage collection and may pick up test executables from unrelated builds and seems out of scope for the stuart command invocation.